packages/modules/src/modules/banners/worldPressFreedomDay/components/SupportCta.tsx

```diff
@@ -57,7 +57,7 @@ export const SupportCta = ({
                     onClickAction={supportUrl}
                     showArrow
                     data-ignore="global-link-styling"
-                    css={[buttonOverrides, cssOverrides]}
+                    cssOverrides={[buttonOverrides, cssOverrides ?? css``]}
```

Comment:
Is it best to keep this inline with moment template SupportCta in case ever use?

Reply:
Apols bit confused on this one